webviz_config/_config_parser.py

```diff
@@ -403,3 +348,182 @@ def plugin_metadata(self) -> Dict[str, dict]:
+            elif self.configuration["menuOptions"]["barPosition"] not in [
+                "left",
+                "top",
+                "right",
+                "bottom",
+            ]:
+                raise ParserError(
+                    f"{terminal_colors.RED}{terminal_colors.BOLD}"
+                    "Invalid option for menuOptions > barPosition: "
+                    f"{self.configuration['menuOptions']['barPosition']}. "
+                    "Please select one of the following options: left, top, right, bottom."
+                    f"{terminal_colors.END}"
+                )
+
+            if "drawerPosition" not in self.configuration["menuOptions"]:
+                self.configuration["menuOptions"]["drawerPosition"] = "left"
+            elif self.configuration["menuOptions"]["drawerPosition"] not in [
+                "left",
+                "right",
+            ]:
+                raise ParserError(
+                    f"{terminal_colors.RED}{terminal_colors.BOLD}"
+                    "Invalid option for menuOptions > drawerPosition: "
+                    f"{self.configuration['menuOptions']['drawerPosition']}. "
+                    "Please select one of the following options: left, right."
+                    f"{terminal_colors.END}"
+                )
+
+            if "initiallyPinned" not in self.configuration["menuOptions"]:
+                self.configuration["menuOptions"]["initiallyPinned"] = False
+            elif not isinstance(
+                self.configuration["menuOptions"]["initiallyPinned"], bool
+            ):
+                raise ParserError(
+                    f"{terminal_colors.RED}{terminal_colors.BOLD}"
+                    "Invalid option for menuOptions > initiallyPinned: "
+                    f"{self.configuration['menuOptions']['initiallyPinned']}. "
+                    "Please select a boolean value: True, False"
+                    f"{terminal_colors.END}"
+                )
+
+            if "showLogo" not in self.configuration["menuOptions"]:
+                self.configuration["menuOptions"]["showLogo"] = True
+            elif not isinstance(self.configuration["menuOptions"]["showLogo"], bool):
+                raise ParserError(
+                    f"{terminal_colors.RED}{terminal_colors.BOLD}"
+                    "Invalid option for menuOptions > initiallyPinned: "
+                    f"{self.configuration['menuOptions']['showLogo']}. "
+                    "Please select a boolean value: True, False"
+                    f"{terminal_colors.END}"
+                )
+        else:
+            self.configuration["menuOptions"] = {}
+            self.configuration["menuOptions"]["barPosition"] = "left"
+            self.configuration["menuOptions"]["drawerPosition"] = "left"
+            self.configuration["menuOptions"]["initiallyPinned"] = False
+            self.configuration["menuOptions"]["showLogo"] = True
```

Comment:
```suggestion
            self.configuration["menuOptions"] = {
                "barPosition": "left",
                "drawerPosition": "left",
                "initiallyPinned": False,
                "showLogo": True,
            }
```

Reply:
Definitely the nicer way to write that code :+1: